src/screens/saved-jobs/index.tsx

```diff
@@ -41,8 +41,14 @@ const SavedJobs = ({navigation}) => {
   };
 
   const getData = async () => {
-    const resp = await callService(ApiMethods.GET, ENDPOINT.GET_MENTORS);
+    const resp = await callService(
+      ApiMethods.GET,
+      `${ENDPOINT.SAVED_JOBS}test.user@gmail.com`,
```

Comment:
Do we need to append an email to the endpoint ?

Reply:
Yes, I think and it's not working yet, so pass it later will check